kiwiapp-public/src/app/(home)/page.tsx

```diff
@@ -1,3 +1,21 @@
+import {
+  WhyRem,
+  HomeBanner,
+  RemAdvantages,
+  TeamSlider,
+  FeaturesBanner,
+  CustomerSatisfaction,
+} from "@/components/home";
+
 export default function Home() {
-  return <div>Home</div>;
+  return (
+    <>
+      <HomeBanner />
+      <RemAdvantages />
+      <WhyRem />
+      <TeamSlider />
+      <FeaturesBanner />
+      <CustomerSatisfaction />
+    </>
```

Comment:
Can we introduce some light animation when the user scrolls? e.g. when they scroll from top to bottom, each section will be lightly faded in and pushed up.